### PR TITLE
Add bootstrap-waitingfor@1.2.4 w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10403
Related issue(s): # #

@cdnjs/intern3 Please help me reviewing this PR. Thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ehpc/bootstrap-waitingfor
 * Official website (optional, not the repository): bootsnipp.com/snippets/featured/quotwaiting-forquot-modal-dialog
 * NPM package url (optional):
 * License and its reference: MIT, [ref](https://github.com/ehpc/bootstrap-waitingfor/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 97
   - Count of forks: 65
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
I think we've discussed about the new conventions about sending new a pull request in private message, there are some earlier works you'll need to do before sending new pull requests.
So I just closed this PR, you can reopen this when you finished it.

**Comment by hare1039** [comment]
@pvnr0082t Please help me reviewing this PR. Thank you.

File: ajax/libs/bootstrap-waitingfor/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bootstrap-waitingfor",
+  "filename": "bootstrap-waitingfor.min.js",
+  "version": "1.2.4",
+  "description": "Waiting for dialog with progress bar for Bootstrap",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ehpc/bootstrap-waitingfor.git"
```
Comment (clairetsai818) [inline ajax/libs/bootstrap-waitingfor/package.json]: @PeterDaveHello Is it better to use `git://` instead of `git+https://`?

Comment (PeterDaveHello) [inline ajax/libs/bootstrap-waitingfor/package.json]: both fine

**Pull request merged**